spec/support/shared_examples_policy_machine.rb

```diff
@@ -621,12 +640,16 @@
         [@u2, @w, @draft_u2], [@u2, @r, @trash_u2], [@u2, @w, @trash_u2]
       ]
 
-      # TODO:  remove the expected privilege below once prohibitions are put in place.
-      # In the example, @u2 is prohibited from writing to @in_u2
-      expected_privileges << [@u2, @w, @in_u2]
-
       assert_pm_privilege_expectations(policy_machine.privileges, expected_privileges)
     end
+
+    it 'can ignore prohibitions' do
```

Comment:
Shouldn't there be another test that when not ignoring prohibitions that <code>scoped_privileges</code> returns the correct result